src/components/SidePanels/OutlinerPanel/OutlinerPanel.tsx

```diff
@@ -80,7 +85,13 @@ const OutlinerStandAlone: FC<OutlinerPanelProps> = ({ regions }) => {
         onGroupingChange={onGroupingChange}
         onFilterChange={onFilterChange}
       />
-      {regions?.regions?.length > 0 ? (
+      {(regions?.regions?.length > 0 && regions?.filter?.length === 0) ? (
```

Comment:
```suggestion
      {(!regions?.regions?.length && regions?.filter?.length) ? (
```
Isn't this a check **if** there are filters but no results?

Reply:
Which property holds the filtered region results, and which holds the active filters?

Reply:
It doesn't work regions.filter.length starts as undefined. If we use this validation it starts as true and it shows the message that there are no items to be shown in this filter. And regions.filter needs to be undefined because when its undefined outliner is showing regions.regions otherwise it will show regions.filter.